src/App/App.js

```diff
@@ -180,7 +177,7 @@ class App extends Component {
             return <MovieInfo
               submitRating={this.submitRating}
               isCurrentUser={this.state.currentUser.id ? true : false}
-              isRated={this.isMovieRated}
+              isRated={this.isMovieRated(match.params.movieId)}
```

Comment:
was this why this test was breaking?

Reply:
Yep - we needed to call the function here so the thing we were passing down didn't need to be resolved, called or anything. This ensures that we have one clean value being passed down instead of a class method